src/Symfony/Bundle/FrameworkBundle/Tests/Kernel/MicroKernelTraitTest.php

```diff
@@ -39,4 +42,21 @@ public function testAsEventSubscriber()
 
         $this->assertSame('It\'s dangerous to go alone. Take this ⚔', $response->getContent());
     }
+
+    public function testRoutingRouteLoaderTagIsAdded(): void
```

Comment:
IMO we don’t use void return type, or?

cc @nicolas-grekas

Reply:
on tests, sure! should be removed

Reply:
I don't mind removing it but for your information there are already 50 tests with the `void` return type in the code. Should we remove them?